_data/simple-icons.json

```
@@ -4549,6 +4549,11 @@
             "source": "https://evernote.com/about-us",
             "guidelines": "https://evernote.com/about-us"
         },
+        {
+            "title": "Excalidraw",
+            "hex": "6965DB",
+            "source": "https://plus.excalidraw.com"
```

```suggestion
            "source": "https://excalidraw.com"
```

Exact same SVG is available in the background of the canvas here. Let's save a few bytes, eh?